plugins/fireeye_hx/icon_fireeye_hx/util/api.py

```diff
@@ -54,6 +55,23 @@ def get_host_id_from_hostname(self, params: dict) -> requests.Response:
     def get_version(self) -> requests.Response:
         return self.call_api(endpoint.version())
 
+    def check_host_quarantine_status(self, agent_id: str) -> requests.Response:
```

Comment:
```suggestion
    def check_host_quarantine_status(self, agent_id: str) -> dict:
```
`.json()` returns a dict